### ISSUE TITLE
A11y_mssql for VSCode_Database Objects_Color contrast: Color contrast when the focus is at Database object with respect to its background is less than 3:1

### ISSUE THREAD

**Issue by khansahab15** [microsoft/vscode-mssql#16929]
[“Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTag:**#A11yTCS; #Win10; #DesktopApp; #BM_mssql for VSCode_Jan21; #WCAG1.4.11; #A11ySev2; #Benchmark; #mssql for VSCode; #Color Contrast Analyzer;

**Environment Details:**
Application Name: mssql for VSCode
Operating System: Win10
Extension Version: 1.10.0

**Repro Steps:**

1. Open Visual studio code 
2. Click on extension button and search for "mssql"
3. Click on install and move to the SQL Server  button
4. Click on Add Connection Button and insert the valid credentials
5. Once the database is added expand the database 
6. Open Color Contrast analyzer and check the color contrast when the focus is at Database object with respect to its background 

**Actual:**
color contrast when the focus is at Database object with respect to its background is 1.568:1

**Expected:**
color contrast when the focus is at Database object with respect to its background should be greater than or equal to 3:1

**User Impact:**
Low vision users will face difficulty in figuring out the focus of the keyboard

**Comment by fsteffi** [comment]
Github Tags:#A11yMAS;

**Comment by alanrenmsft** [comment]
@fsteffi  @khansahab15 

as shown in screenshots below, our icons have a white border to differentiate it from the background color, the contrast should be enough

![image](https://user-images.githubusercontent.com/13777222/148617134-13417210-bb92-4a4a-af08-90d60c4c68ed.png)

![image](https://user-images.githubusercontent.com/13777222/148617169-da006a1d-ea2f-4756-9919-30dafe0adbe0.png)

If you don't agree, please include screenshots and theme name and explanation so that I don't have to guess.

**Comment by Yashi1201** [comment]
@alanrenmsft Here, in the dark theme, issue is not repro, the color contrast ratio when focus is at database objects w.r.t background is 3.636:1
Version: 1.75.0-insider (user setup)
![16929_Dark theme_issue fixed](https://user-images.githubusercontent.com/93374043/219311414-51e1361e-8f42-4ca4-9e8c-2ea15e55d7cc.png)

But for the users who uses light theme, issue is still repro, the color contrast ratio when focus is at database objects w.r.t background is 1.679:1 which is less than 3:1
Version: 1.75.0-insider (user setup)
![16929_Light theme_Issue repro](https://user-images.githubusercontent.com/93374043/219312019-8900c892-b8ed-47df-ab36-6328748dc6f5.png)

So, could you please reopen the bug and fix the issue. Bugs can be closed by Trusted Tester after bug verification.
**GithubTags**:#Regressed:02-16-23;

**Comment by alanrenmsft** [comment]
@Yashi1201 for the light theme, where are you picking the color from, the blue you have is much lighter than the actual background.

**Comment by Yashi1201** [comment]
@alanrenmsft In the light theme, when keyboard focus is at selected database object i.e "DWDiagonistics..", the background color is the grey color and foreground color is of the focus boundary i.e light blue that is coming when keyboard focus is at database object.
**GithubTags:**#Regressed:02-16-23;

**Comment by sspatil123** [comment]
The issue was fixed in Version: 1.77.0-insider (user setup) issue is fixed. 
GithubTags:#Regressed:03-20-23;#Closed;
![16929_Fixed](https://user-images.githubusercontent.com/85863999/226293947-4aea680d-806a-45c3-9c75-afb64aad6a93.png)
